File: FrontEnd/Prueba/src/app/models/pedido_impuesto.ts
export class Pedido_Impuesto
{
  porcentaje: number;
  totalImpCobrado: number;

  constructor(porcentaje: number, totalImpCobrado: number)
  {
    this.porcentaje = porcentaje;
    this.totalImpCobrado = totalImpCobrado;
  }
}

File: BackEnd/demo/src/main/java/com/Crisalis/demo/model/Pedido_impuesto.java
package com.Crisalis.demo.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.math.BigDecimal;

@Entity
@Table(name = "Pedido_Impuesto")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Pedido_impuesto {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ID")
    private Integer peIm_ID;
    @Column(name = "Porcentaje")
    private Integer imp_Porcentaje;
    @Column(name = "TotalImpuestoCobrado")
    private BigDecimal imp_TotalImpCobrado;
    @OneToOne
    //@JoinColumn(name = "imp_ID")
    private Impuesto impuesto;
}
